.gitignore

```diff
@@ -9,7 +9,6 @@
 
 # quarto build artefacts
 .jupyter_cache
-*.ipynb
```

Comment:
I would request to only exclude the one particular notebook `analyses/verification.ipynb` from being ignored, while all other Jupyter notebooks should kept being ignored. The reason for this is that I and maybe also some of you have notebooks for testing, which then would always show up as not tracked when running `git status`.
One way to achieve this would be to leave `.gitignore` as is and force `git add/commit` the new notebook `analyses/verification.ipynb`.

Reply:
Thanks Henning! Just uploaded a new .gitignore. This now ignores all files ".ipynb" files except for those in the analysis folder.

Reply:
Thanks! :)